plugins/language_st.lua

```diff
@@ -0,0 +1,125 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+-- Language syntax references
+-- https://pdhonline.com/courses/e334/e334content.pdf
+
+syntax.add {
+  name = "PLC Structured Text IEC 61131-3",
+  files = { "%.stx?$", "%.iecst$" },
+  comment = "//",
+  block_comment = { "%(%*", "%*%)" },
+  patterns = {
+    { pattern = "//.*",                                     type = "comment"  }, -- Single-line comment
+    { pattern = { "%(%*", "%*%)" },                         type = "comment"  }, -- Multi-line comment
+    { pattern = { '"', '"', '\\' },                         type = "string"   }, -- String, quotation marks
+    { pattern = { "'", "'", '\\' },                         type = "string"   }, -- String, apices
+    { pattern = "%w+#[0-9]+m?s?",                           type = "number"   }, -- Time/Date formats
+    { pattern = "[%a_][%w_]*%f[(]",                         type = "function" }, -- Function
+    { pattern = "[%a_][%w_]*",                              type = "symbol"   }, -- Symbols
+    { regex   = "\\:\\s*(?:\\w+_?)+",                       type = "keyword2" }, -- Variable/Method Type
```

Comment:
Would
```suggestion
    { pattern = ":%s*%w[%w_]*",                             type = "keyword2" }, -- Variable/Method Type
```
be equivalent?

Reply:
Yes, thanks!